### ISSUE TITLE
Deprecation warnings when using Rails 5.2 beta

### ISSUE THREAD

**Issue by mrgilman** [Casecommons/pg_search#374]
I upgraded a Rails 5.1.4 application to Rails 5.2.0.beta2 and noticed deprecation warnings whenever pg_search methods were being called:

> DEPRECATION WARNING: Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "pg_search_5ff6d794e372fc33089600.rank DESC, \"posts\".\"id\" ASC". Non-attribute arguments will be disallowed in Rails 6.0. This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passedby wrapping them in Arel.sql(). (called from call at /app/app/graphql/resolvers/all_posts.rb:11)

This appears to be triggered by an upcoming feature of Rails 6 that requires raw SQL fragments to be explicitly marked, see [the ActiveRecord changelog](https://github.com/rails/rails/blob/master/activerecord/CHANGELOG.md#rails-520beta1-november-27-2017).

Thanks for your hard work on this gem, we love using it!

**Comment by kddeisz** [comment]
I believe https://github.com/Casecommons/pg_search/pull/376 will suffice for this.

**Comment by kddeisz** [comment]
If anyone is looking for this in the meantime, adding

```ruby
module PgSearchFlexibleOrderMethod
  module FlexibleOrder
    def order(value)
      value.is_a?(String) ? super(Arel.sql(value)) : super(value)
    end
  end

  def apply(scope)
    modified =
      scope.all.spawn.tap do |new_scope|
        new_scope.class_eval { prepend FlexibleOrder }
      end

    super(modified)
  end
end

class PgSearch::ScopeOptions
  prepend PgSearchFlexibleOrderMethod
end
```

into an initializer should do the trick until the next version is out.

**Comment by nertzy** [comment]
Fixed by #376
